services/director-v2/src/simcore_service_director_v2/modules/rabbitmq.py

```diff
@@ -27,9 +28,17 @@ async def handler_out_of_credits(app: FastAPI, data: bytes) -> bool:
     settings: AppSettings = app.state.settings
 
     if (
-        settings.DYNAMIC_SERVICES.DYNAMIC_SCHEDULER.DIRECTOR_V2_DYNAMIC_SCHEDULER_IGNORE_SERVICES_SHUTDOWN_WHEN_CREDITS_LIMIT_REACHED
+        settings.DYNAMIC_SERVICES.DYNAMIC_SCHEDULER.DIRECTOR_V2_DYNAMIC_SCHEDULER_CLOSE_SERVICES_VIA_FRONTEND_WHEN_CREDITS_LIMIT_REACHED
     ):
-        _logger.debug("Skipped shutting down services for wallet %s", message.wallet_id)
+        _logger.warning(
+            "Shutting down (via frontend) service: '%s' for user '%s' because wallet '%s' is out of credits.",
```

Comment:
why do we care about the frontend here? it sounds like the frontend requested it but it's not the case right? it's just the credits

Reply:
Correct. We care about this situation for the following reasons:
- backend (resource usage tracker) asks to shut down the service
- this is a temporary fix, so I will leave the warning here since, in the future this should go away and only the else case should exist in the future.

Reply:
what I mean is that this message is confusing. via means it comes through the frontend and that is not the case.

Reply:
OK I've renamed it a bit.